docs/source/daml-integration-kit/index.rst

```diff
@@ -0,0 +1,329 @@
+.. _integration-kit:
+
+DAML Integration Kit - PRE-ALPHA
+################################
+
+The DAML Integration Kit allows third-party ledger developers to
+implement a DAML Ledger on top of their distributed-ledger or database of
+choice. How does that work?
+
+A DAML Ledger implementation is a server serving the
+:doc:`/app-dev/ledger-api-introduction/index` as per the semantics defined in
+the :doc:`/concepts/ledger-model/index` and the
+`DAML-LF specification <https://github.com/digital-asset/daml/blob/master/daml-lf/spec/daml-lf-1.rst>`_.
+Theoretically, you could
+implement such a server from scratch following the above documentation.
+Practically, we though strongly recommend to use our integration kit and follow
+the guides below for
+:ref:`implementing <integration-kit_implementing>`,
+:ref:`deploying <integration-kit_deploying>`,
+:ref:`testing <integration-kit_testing>`, and
+:ref:`benchmarking <integration-kit_benchmarking>` your own DAML Ledger
+server. This way you can focus on your distributed-ledger or database of
+choice and reuse our DAML Ledger server and DAML interpreter code for
+implementing the DAML Ledger API.
+
+
+.. _integration-kit_status-and_roadmap:
+
+Status and Roadmap
+******************
+
+The current status of the integration kit is PRE-ALPHA. We expect it to evolve
+to ALPHA soon, BETA later, and GA quite a bit later. The roadmap below
+explains what we mean by these different statuses, and what's missing to
+progress.
+
+PRE-ALPHA (current status)
+  We develop the DAML Integration Kit in an open-first model. The PRE-ALPHA
+  status means that there is enough information for interested third-parties
+  to follow the *development* of the DAML integration kit. **We do though not
+  make any stability guarantees!** Moreover the documentation is expected to
+  contain holes, which are usually linked to GitHub issues.
+
+ALPHA (`tracking GitHub issue <https://github.com/digital-asset/daml/issues/658>`__)
+  Once we reach ALPHA status, the DAML Integration Kit is ready to be used by
```

Comment:
I would suggest slightly less self-debilitating tone:
"Once ALPHA status is reached, the DAML Integration Kit is ready to be used by
third-parties willing to accept following caveats:"